# Generated for NaoQi method ALFrameManager.onRubyPrint
# Print Ruby message in Choregraphe or logger. Should not be called manually.

# Message to print
string message
---


